common/test/java/com/couchbase/lite/LiveQueryTest.java

```diff
@@ -129,21 +129,16 @@ public void testLiveQueryDelay() throws CouchbaseLiteException, InterruptedExcep
             });
 
         try {
```

Comment:
Doesn't the first notification still take 200 ms?  Is this going to make the test flaky?